rockspecs/sqlite-lua-master-0.rockspec

```diff
@@ -0,0 +1,33 @@
+rockspec_format = "3.0"
+package = "sqlite-lua"
+version = "master-0"
+source = {
+  url = "git://github.com/tami5/sql.nvim",
+  branch = "master",
+}
+
+dependencies = {
+  "lua >= 5.1",
```

Comment:
dependency is wrong this only works with luajit because lua >= 5.1 doesnt have ffi.

I dont know if you can specify this but lua 5.1 is also valid if https://github.com/facebookarchive/luaffifb is installed. but that needed to be tested because https://github.com/facebookarchive/luaffifb is not 100% compatible

Reply:
Also dont you have luv as dependency somewhere?

Reply:
Yep lua 5.1 is wrong, I'll just add luv. Thanks conni